tests/008-test-default-lv-data-size.sh

```diff
@@ -0,0 +1,84 @@
+source $SRCDIR/libtest.sh
+
+# Default size of LV that is created from the volume group 
+# by docker-storage-setup is 40% of the free space of the
+# VG, i.e DATA_SIZE=40%FREE. If dss gets modified and changes
+# this default behavior, below code should be modified to 
+# reflect changes
+test_default_lv_size() {
+  local devs=$TEST_DEVS # devices set by user in dss-config
+  local test_status
+  local testname=`basename "$0"`
+  local vg_name
+  local vg_name="dss-test-foo"
+  local pe_total
+  local pe_free
+  local pe_used_percentage
+  
```

Comment:
I think you should be able to put local variable names with spaces on same line instead of having separate line for every variable.